security/login-error.php

```diff
@@ -10,11 +10,11 @@
  * 
  * @since 1.1
  */
-function use_ambiguous_login_error( string $error ): string {
+function use_ambiguous_login_error( $error ): string {
 	global $errors;
 
 	if ( ! is_wp_error( $errors ) ) {
-		return $error;
+		return (string) $error;
```

Comment:
If we are passed something other than a string, limit possible damage by explicitly casting the argument to a string.

Reply:
@sjinks Is this something we want to expose though since it gets directly outputted?